static/chat_db_migrations/1562227874_add_device_acks_metadata.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE device_not_found_acks (
```

Comment:
Can we add a comment here what this table contains? Like: "it contains information to which identities and when we send info about our devices".